pal/audio/web/player-web.ts

```diff
@@ -438,7 +440,9 @@ export class AudioPlayerWeb implements OperationQueueable {
         try {
             if (this._sourceNode) {
                 this._sourceNode.stop();
+                this._sourceNode.disconnect();
                 this._sourceNode.buffer = null;
+                this._sourceNode = null as any;
```

Comment:
maybe we should change the type of  _sourceNode from `AudioBufferSourceNode | undefined` to `AudioBufferSourceNode | null`

Reply:
set this._sourceNode to undefined